certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+          "$ref": "#/definitions/ResourceResponse"
+        }
+      }
+    },
+    "SourceType": {
+      "enum": [
+        "Ppmx",
+        "O365Group",
+        "VSTS",
+        "Jira",
+        "File",
+        "O365Planner",
+        "O365User",
+        "Spo",
+        "O365TeamsChannel",
+        "PpmxTime"
+      ],
+      "type": "string",
+      "x-ms-summary": "Source type",
+      "description": "Type of source"
+    },    
+    "BaselineData": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "startDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "dueDate": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "duration": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "effort": {
+          "format": "double",
+          "type": "number"
+        },
+        "originalEstimate": {
+          "format": "double",
+          "type": "number"
+        }
+      }
+    },
+    "TaskAttributes": {
+      "x-ms-summary": "Task fields",
+      "description": "Task fields (e.g. {\"Name\": \"My Name\", \"Description\": \"TBD\"})",
+      "type": "object",
+      "properties": {},
+      "additionalProperties": {}
+    },
+    "TaskResponse": {
+      "description": "Task details",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "Task Id",
+          "description": "Task Id",
+          "type": "string"
+        },
+        "attributes": {
+          "$ref": "#/definitions/TaskAttributes"
+        },
+        "sourceType": {
+          "$ref": "#/definitions/SourceType"
+        },
+        "baseline": {
+          "$ref": "#/definitions/BaselineData"
+        }
+      }
+    },    
+    "TaskResponseListTask": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "x-ms-summary": "Tasks",
+          "description": "List of tasks",
+          "items": {
+            "$ref": "#/definitions/TaskResponse"
+          }
+        }
+      }
+    },
+    "TaskResponseResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/TaskResponse"
+        }
+      }
+    },
+    "UpdateChallengeRequest": {
+      "description": "Challenge Update model",
+      "type": "object",
+      "properties": {
+        "updates": {
+          "$ref": "#/definitions/ChallengeAttributes"
+        }
+      }
+    },
+    "UpdateIdeaRequest": {
+      "description": "Idea Update model",
+      "type": "object",
+      "properties": {
+        "updates": {
+          "$ref": "#/definitions/IdeaAttributes"
+        }
+      }
+    },
+    "TokenInfo": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "x-ms-summary": "Token Id",

```

Comment:
@vsorokin-ppmexpress please don't copy and paste summaries and descriptions, try to make them unique. You can refer to the [English language strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings). Please look through these properties and fix them.